.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 orbs:
   aws-cli: circleci/aws-cli@1.3.0
   jq: circleci/jq@1.9.1
-  utils: coda/utils@0.5.0
+  utils: coda/utils@0.4.0
```

Comment:
There's a slight regression with `utils/slack-notify-waiting-for-approval` in `0.5.0` that had an encoding error [causing the notification job to silently fail](https://app.circleci.com/pipelines/github/coda/packs-sdk/5422/workflows/1eda385a-9574-4ffb-ad8b-0e4e004103e1/jobs/5555)

Reply:
Can you bump this to the latest version [coda/utils@2.0.0](https://circleci.com/developer/orbs/orb/coda/utils)?